### PR TITLE
Update register() to use args and kwargs

### PR DESCRIPTION
**Pull request by imbatraman**
Updating def register() to *args and **kwargs. args for pkg_root and kwargs for remote

TODO:
- test
- update doc strings

### CODE REVIEW

**Comment by kennyworkman** [comment]
hey @imbatraman can you provide more context on this change?

**Comment by imbatraman** [comment]
> 

hey @kennyworkman - this one relates to my chat with @ayushkamat on updating `latch register` to work on either the current working dir (if no args are provided) or the dir provided

Here's an extract from the chat:

> add the ability to run latch register  without any arguments and have that run latch register on the current working directory 
> right now, a call to latch register requires the user to specify which directory they want to be registered, and requires the user to run the command while outside the directory in question

**Review by ayushkamat — CHANGES_REQUESTED** [review]

File: latch_cli/services/register/register.py [hunk 1]
```diff
@@ -68,8 +68,8 @@ def _print_reg_resp(resp, image):
 
 
 def register(
```
[the remark this answers is not in the document]
Comment (imbatraman) [reply latch_cli/services/register/register.py:70]: sure thing - i'll change it there

**Review by ayushkamat — CHANGES_REQUESTED** [review]
One small thing: change the @click.argument decorator for the function so that the pkg_root argument isn't required - see the Click documentation for how to do this

**Pull request closed without merging**